app/src/main/java/com/movies/domain/usecase/movies/GetMoviesUseCase.kt

```diff
@@ -10,9 +10,9 @@ class GetMoviesUseCase(
 ) : BaseUseCase<CategoryType, List<MovieDomainModel>>() {
 
     override suspend fun invoke(params: CategoryType?): List<MovieDomainModel> {
-        return params?.let {
-            moviesRepository.fetchMovies(it)
-        } ?: emptyList()
+        return params.let {
+            moviesRepository.fetchMovies(it!!)
+        }
     }
 
```

Comment:
empty

Reply:
there should be, it makes more readable where the class end